src/sentry/static/sentry/app/views/settings/projectAlerts/issueEditor/ticketRuleForm.tsx

```diff
@@ -192,7 +172,8 @@ class TicketRuleForm extends React.Component<Props, State> {
   };
 
   render() {
-    const {ticketType, link} = this.buildText();
+    const {ticketType, link} = this.props.instance || {};
+
     const text = t(
       'When this alert is triggered a %s will be created with the following fields. ',
```

Comment:
this is where I meant it should be a or an depending on whether it's Jira or Azure, sorry

Reply:
@ceorourke Not sure what you mean here. We should keep all integration specific logic on the backend.

Reply:
@scefali not because it's a particular integration, but because one starts with a consonant and one starts with a vowel

Reply:
ohhh yea i see what you mean, let me fix that